File: 1894-merge-strings-alternately/merge-strings-alternately.cpp
class Solution {
public:
    string mergeAlternately(string word1, string word2) {
        int i=0,j=0;
        string neww="";
        while(i<word1.size()||j<word2.size()){
            if(i<word1.size())
            neww+=word1[i++];
            if(j<word2.size())
            neww+=word2[j++];
        }
        return neww;
    }
};

File: 1894-merge-strings-alternately/merge-strings-alternately.py
class Solution(object):
    def mergeAlternately(self, word1, word2):
        result=""
        n=len(word1)
        m=len(word2)
        siz=max(n,m)
        for i in range(siz):
            if i<n:
             result+=word1[i]
            if i<m:
             result+=word2[i]
             
        return result
        